src/index.spec.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
```

Comment:
Another non-blocking comment: I was initially confused when I didn't see a matching "index.js" file in the same directory. Only thing I could think of was to put another index.js file in this directory (which has all the logic of the root's index.js file), and simply require it in the root index.js file. Not a deal breaker...